### PR TITLE
Partial fix for #600 - incorrect localized messages

### PR DESCRIPTION
**Pull request by wizofaus**
Need translators to provide proper messages for MinimumLengthValidator & MaximumLengthValidator for most languages.

### CODE REVIEW

File: src/FluentValidation/Resources/Languages/GeorgianLanguage.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ namespace FluentValidation.Resources {
 
 	internal class GeorgianLanguage : Language {
 
-		public override string Name => "ge";
+		public override string Name => "ka";
```
Comment (JeremySkinner) [inline src/FluentValidation/Resources/Languages/GeorgianLanguage.cs]: Good catch

File: src/FluentValidation.Tests/LanguageManagerTests.cs [hunk 2]
```diff
@@ -101,6 +101,32 @@ public void Gets_translation_for_croatian_culture()
...
+		public void All_localizatons_have_same_parameters_as_English() {
+
+			LanguageManager manager = (LanguageManager)_languages;
+			var languages = manager.GetSupportedLanguages();
+			var keys = manager.GetSupportedTranslationKeys();
+			
+			Assert.All(languages, l => Assert.All(keys, k => Check_Parameters_Match(l, k)));
+		}
+
+		void Check_Parameters_Match(string languageCode, string translationKey) {
```
Comment (JeremySkinner) [inline src/FluentValidation.Tests/LanguageManagerTests.cs]: Rename to CheckParametersMatch

File: src/FluentValidation/Resources/Languages/ChineseSimplifiedLanguage.cs [hunk 3]
```diff
@@ -27,12 +27,12 @@ internal class ChineseSimplifiedLanguage : Language {
...
 			Translate<LengthValidator>("'{PropertyName}' 的长度必须在 {MinLength} 到 {MaxLength} 字符，您已经输入了 {TotalLength} 字符。");
-			Translate<MinimumLengthValidator>("'{PropertyName}' 的长度必须在 {MinLength} 到 {MaxLength} 字符，您已经输入了 {TotalLength} 字符。");
-			Translate<MaximumLengthValidator>("'{PropertyName}' 的长度必须在 {MinLength} 到 {MaxLength} 字符，您已经输入了 {TotalLength} 字符。");
+			Translate<MinimumLengthValidator>("'{PropertyName}' 的长度必须在 {MinLength} 到 1000 字符，您已经输入了 {TotalLength} 字符。");
+			Translate<MaximumLengthValidator>("'{PropertyName}' 的长度必须在 0 到 {MaxLength} 字符，您已经输入了 {TotalLength} 字符。");
 			Translate<LessThanOrEqualValidator>("'{PropertyName}' 必须小于或等于 '{ComparisonValue}'。");
 			Translate<LessThanValidator>("'{PropertyName}' 必须小于 '{ComparisonValue}'。");
 			Translate<NotEmptyValidator>("请填写 '{PropertyName}'。");
-			Translate<NotEqualValidator>("'{PropertyName}' 不能和 '{PropertyValue}' 相等。");
+			Translate<NotEqualValidator>("'{PropertyName}' 不能和 '{ComparisonValue}' 相等。");
```
Comment (JeremySkinner) [inline src/FluentValidation/Resources/Languages/ChineseSimplifiedLanguage.cs]: Good catch

File: src/FluentValidation.Tests/LanguageManagerTests.cs [hunk 4]
```diff
@@ -101,6 +101,32 @@ public void Gets_translation_for_croatian_culture()
 		}
 
 
+		[Fact]
```
Comment (JeremySkinner) [inline src/FluentValidation.Tests/LanguageManagerTests.cs]: Should probably make this a [Fact(Skip="Explicit")] for now as we know it's going to fail until the messages are re-translated.

Comment (wizofaus) [reply src/FluentValidation.Tests/LanguageManagerTests.cs]: As of this PR they shouldn't fail though.  Are you seeing failures?
Sure, hard-coding 1000 isn't great but as it is it shows -1 which is surely worse...
I could hard-code an infinity symbol instead!
All those others issues with the translations were picked up by the test.
BTW there is a potential bug in my code if you happened to have a message that had 3 consecutive {'s - not sure if you're really worried about that possibility though.

Comment (wizofaus) [reply src/FluentValidation.Tests/LanguageManagerTests.cs]: Ready to push a new commit if you've made a call on the use 1000 (I did even try replacing with ∞ : not sure how, e.g.  "'Code' doit contenir entre 2 et ∞ caractères" would read to a French speaker though!)

Comment (JeremySkinner) [reply src/FluentValidation.Tests/LanguageManagerTests.cs]: @wizofaus Yes you were right, the test does pass (I'd modified something when I was testing).

Let's go with the 1000 characters

**Comment by wizofaus** [comment]
Hmm, somehow a rogue commit ended up this branch, will have to re-do PR.

**Comment by JeremySkinner** [comment]
I've merged this into the master branch with 1 changes.

The CheckParametersMatch method does not build in the FluentValidation.Tests.NetCoreapp1 test project as .net core 1.0 doesn't include the CultureInfo.CreateSpecificCulture method. I've modified the method to use the `new CultureInfo(languageCode)` constructor instead as this works on .net core 1, .net core 2 and regular framework.  

This will be in the 7.3 beta which I'll try and push to nuget shortly.

**Pull request closed without merging**